### DISCUSSION TITLE
How to check balance of a token?

### DISCUSSION THREAD

**Discussion by grandwiz** [WalletConnect/WalletConnectUnity#38] [Q&A]
Hi, I migrated my project from Chainlink SDK to WalletConnect;

I am trying to look for a function similar to:

`ERC1155.BalanceOf(chain, network, contract, account, token);`

to return the balance of a token the connected user has. I cannot access the docs on the main website due to an SSL error.

**Comment by ecp4224** [comment]
Are you using NEthereum in your project or just WalletConnect?
Sadly, the current version of the project only supports write transactions and does not support read transactions. If you are using NEthereum, you can use the [WalletCoonnectSharp.NEthereum](https://www.nuget.org/packages/WalletConnect.NEthereum/1.7.1) extension to add read operation support. You will need access to an RPC node
